AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx

```diff
@@ -72,7 +90,23 @@ export default class LocationForm extends React.Component {
     }
 
     onSiteChange(event) {
-        this.setState({ siteName: event.target.value });
+        if(event.target.value.length < 200){
+            this.setState({ siteName: event.target.value });
+        }
+    }
+
+    onLongitudeChange(event) {
+        if(event.target.value < 100 && event.target.value > -100){
+                const newLong = event.target.value === '' ? null : event.target.value //This replaces empty strings with null.
```

Comment:
I'm not a fan of this comment appearing in lots of places, suggests we should be more comfy with the syntax :(